documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -607,6 +607,45 @@ The REST connector differs from the above connectors because it piggybacks on th
 
 ----
 
+==== Multi-tenant router
```

Comment:
Probably a bit late in the game  :smile:,  and apologies if it was already answered elsewhere, but why a router is needed? In the design doc it says "The problematic part is Infinispan Endpoint, which expects a single cache-container per server." 
What were the drawbacks of changing it so that an endpoint could be capable of serving multiple cache containers?